pkg/model/manifest.go

```diff
@@ -236,15 +236,15 @@ func GetManifestV2(manifestPath string) (*Manifest, error) {
 		return nil, err
 	}
 	if inferredManifest != nil {
-		if devManifest != nil {
-			inferredManifest.mergeWithOktetoManifest(devManifest)
-		} else if inferredManifest.Type == StackType {
+		if inferredManifest.Type == StackType {
 			inferredManifest, err = inferredManifest.InferFromStack()
 			if err != nil {
 				return nil, err
 			}
-			//TODO: uncomment
-			// inferredManifest.Deploy.Compose.Stack.Endpoints = devManifest.Deploy.Endpoints
+			inferredManifest.Deploy.Compose.Stack.Endpoints = devManifest.Deploy.Endpoints
```

Comment:
I am getting a segmentation fault error here when running `okteto build` in this repo:
https://github.com/pchico83/compose-getting-started

Reply:
I can't repro it, are you using any flag on the build command?

Reply:
Nope, just `okteto build` or `okteto deploy`. Let me try again